input-output-hk/lace
# [LW-7570]: update handle when sending tokens

# Checklist

- [ ] JIRA - LW-7570

---

## Proposed solution

Explain how does this PR solves the problem stated in JIRA ticket.
You can also enumerate different alternatives considered while approaching this task.

## Testing

1. Add one of your own handles to the address book and send it to another address (can skip if a previously saved handle has changed hands).
2. Go to send flow and proceed to add this handle from the address book.
3. Expected: A warning with a review option should appear post-verification. 
4. Click the call to action and you should be taken to an update/delete address flow.
5. Take an action - Update or Delete.
6. Expected: You should be taken back to the address input screen.

## Screenshots


https://github.com/input-output-hk/lace/assets/27725122/41923d92-4bec-469f-bd68-77f1cec88cbe




---
# Allure report
`allure-report-publisher` generated test report!



**smokeTests**: ❌ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/linux/chrome/1345/5783623371/index.html) for [4b46e913](https://github.com/input-output-hk/lace/pull/355/commits/4b46e9138890373dda973194632b7261013f5699)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 35     | 1      | 0       | 0     | 36    | ❌     |
